### PR TITLE
decompile a little more of player_pc.c and dump half of battle_anim data incbins

### PR DESCRIPTION
**Pull request by ProjectRevoTPP**

### CODE REVIEW

File: data/battle_anim_80CA710.s [hunk 1]
```diff
@@ -1303,7 +1303,13 @@ gSpriteAffineAnim_83D7250:: @ 83D7250
 	obj_rot_scal_anim_end
 
 	.align 2
-	.incbin "baserom.gba", 0x003d7278, 0x48
+@ unused tilemap?
```
Comment (marijnvdwerf) [inline data/battle_anim_80CA710.s]:
```
        obj_rot_scal_anim_frame 16, 16, 0, 0
        obj_rot_scal_anim_frame 30, 30, 0, 8
        obj_rot_scal_anim_frame 0, 0, 0, 16
        obj_rot_scal_anim_loop 0
        obj_rot_scal_anim_frame 0, 0, 4, 11
        obj_rot_scal_anim_frame 0, 0, -4, 11
        obj_rot_scal_anim_loop 2
        obj_rot_scal_anim_frame -30, -30, 0, 8
        obj_rot_scal_anim_end 0
```

File: data/battle_anim_80CA710.s [hunk 2]
```diff
@@ -3730,10 +3849,49 @@ gBattleAnimSpriteTemplate_83DA88C:: @ 83DA88C
...
+	.2byte 0x7FFF
+	.2byte 0x0000
+	.byte 0x00
+	.byte 0x00
+	.byte 0x00, 0x00 @ struct padding
 
+	.align 2
 gUnknown_083DA8C4:: @ 83DA8C4
-	.incbin "baserom.gba", 0x003da8c4, 0x18
+	.2byte 0x0040
```
Comment (marijnvdwerf) [inline data/battle_anim_80CA710.s]: obj_rot_scal_anim_frame 64, -4, 0, 20
        obj_rot_scal_anim_frame 0, 0, 0, -56
        obj_rot_scal_anim_end 0

File: data/battle_anim_80CA710.s [hunk 3]
```diff
@@ -3730,10 +3849,49 @@ gBattleAnimSpriteTemplate_83DA88C:: @ 83DA88C
 
 	.align 2
 gUnknown_083DA8A4:: @ 83DA8A4
-	.incbin "baserom.gba", 0x003da8a4, 0x20
+	.2byte 0xFFF8
```
Comment (marijnvdwerf) [inline data/battle_anim_80CA710.s]: obj_rot_scal_anim_frame -8, 10, 0, 16
        obj_rot_scal_anim_frame 18, -18, 0, 16
        obj_rot_scal_anim_frame -20, 16, 0, 8
        obj_rot_scal_anim_end 0

File: data/battle_anim_80CA710.s [hunk 4]
```diff
@@ -1981,7 +2077,29 @@ gBattleAnimSpriteTemplate_83D7C90:: @ 83D7C90
 
 	.align 2
 gUnknown_083D7CA8:: @ 83D7CA8
-	.incbin "baserom.gba", 0x003d7ca8, 0x20
+	.2byte 0xFFF4
```
Comment (marijnvdwerf) [inline data/battle_anim_80CA710.s]: obj_rot_scal_anim_frame -12, 8, 0, 4
	obj_rot_scal_anim_frame 20, -20, 0, 4
	obj_rot_scal_anim_frame -8, 12, 0, 4
	obj_rot_scal_anim_end 0

File: data/battle_anim_80CA710.s [hunk 5]
```diff
@@ -1769,7 +1855,17 @@ gBattleAnimSpriteTemplate_83D79A4:: @ 83D79A4
 
 	.align 2
 gUnknown_083D79BC:: @ 83D79BC
-	.incbin "baserom.gba", 0x003d79bc, 0x10
+	.2byte 0x0060
```
Comment (marijnvdwerf) [inline data/battle_anim_80CA710.s]: obj_rot_scal_anim_frame 96, -13, 0, 8
	obj_rot_scal_anim_end 0

File: data/battle_anim_80CA710.s [hunk 6]
```diff
@@ -1651,7 +1703,41 @@ gBattleAnimSpriteTemplate_83D7798:: @ 83D7798
 
 	.align 2
 gUnknown_083D77B0:: @ 83D77B0
-	.incbin "baserom.gba", 0x003d77b0, 0x30
+	.2byte 0xFFF6
```
Comment (marijnvdwerf) [inline data/battle_anim_80CA710.s]: obj_rot_scal_anim_frame -10, 9, 0, 7
	obj_rot_scal_anim_frame 20, -20, 0, 7
	obj_rot_scal_anim_frame -20, 20, 0, 7
	obj_rot_scal_anim_frame 10, -9, 0, 7
	obj_rot_scal_anim_loop 2
	obj_rot_scal_anim_end 0

File: data/battle_anim_80CA710.s [hunk 7]
```diff
@@ -1604,11 +1610,57 @@ gBattleAnimSpriteTemplate_83D76DC:: @ 83D76DC
...
+	.2byte 0x7FFF @ terminator?
+	.2byte 0x0000
+	.byte 0x00
+	.byte 0x00
+	.byte 0x00, 0x00 @ struct padding
 
+	.align 2
 gUnknown_083D7714:: @ 83D7714
-	.incbin "baserom.gba", 0x003d7714, 0x20
+	.2byte 0xFFFC
```
Comment (marijnvdwerf) [inline data/battle_anim_80CA710.s]: obj_rot_scal_anim_frame -4, -5, 0, 12
	obj_rot_scal_anim_frame 0, 0, 0, 24
	obj_rot_scal_anim_frame 4, 5, 0, 12
	obj_rot_scal_anim_end 0

File: data/battle_anim_80CA710.s [hunk 8]
```diff
@@ -1604,11 +1610,57 @@ gBattleAnimSpriteTemplate_83D76DC:: @ 83D76DC
 	spr_template 10138, 10138, gOamData_837E054, gSpriteAnimTable_83D76D4, NULL, gDummySpriteAffineAnimTable, sub_80D0228
 
 	.align 2
+	@ probably 4 structs of Struct_0202F7D4.
 gUnknown_083D76F4:: @ 83D76F4
-	.incbin "baserom.gba", 0x003d76f4, 0x20
+	.2byte 0xFFFA
```
Comment (marijnvdwerf) [inline data/battle_anim_80CA710.s]: obj_rot_scal_anim_frame -6, 4, 0, 8
	obj_rot_scal_anim_frame 10, -10, 0, 8
	obj_rot_scal_anim_frame -4, 6, 0, 8
	obj_rot_scal_anim_end 0

File: data/battle_anim_812C144.s [hunk 9]
```diff
@@ -597,4 +841,38 @@ gBattleAnimSpriteTemplate_8402B10:: @ 8402B10
 
 	.align 2
 gUnknown_08402B28:: @ 8402B28
-	.incbin "baserom.gba", 0x00402b28, 0x30
+	.2byte 0x0000
```
Comment (marijnvdwerf) [inline data/battle_anim_812C144.s]: obj_rot_scal_anim_frame 0, 16, 0, 4
	obj_rot_scal_anim_frame -2, 0, 0, 8
	obj_rot_scal_anim_frame 0, 4, 0, 4
	obj_rot_scal_anim_frame 0, 0, 0, 24
	obj_rot_scal_anim_frame 1, -5, 0, 16
	obj_rot_scal_anim_end 0

File: data/battle_anim_812C144.s [hunk 10]
```diff
@@ -521,7 +749,23 @@ gBattleAnimSpriteTemplate_84029C4:: @ 84029C4
 
 	.align 2
 gUnknown_084029DC:: @ 84029DC
-	.incbin "baserom.gba", 0x004029dc, 0x18
+	.2byte 0x0000
```
Comment (marijnvdwerf) [inline data/battle_anim_812C144.s]: obj_rot_scal_anim_frame 0, -16, 0, 6
	obj_rot_scal_anim_frame 0, 16, 0, 6
	obj_rot_scal_anim_end 0

File: data/battle_anim_812C144.s [hunk 11]
```diff
@@ -457,12 +662,35 @@ gBattleAnimSpriteTemplate_8402894:: @ 8402894
 
 	.align 2
 gUnknown_084028AC:: @ 84028AC
-	.incbin "baserom.gba", 0x004028ac, 0x20
+	.2byte 0xFFF0
```
Comment (marijnvdwerf) [inline data/battle_anim_812C144.s]: obj_rot_scal_anim_frame -16, 16, 0, 6
	obj_rot_scal_anim_frame 16, -16, 0, 12
	obj_rot_scal_anim_frame -16, 16, 0, 6
	obj_rot_scal_anim_end 0

File: data/battle_anim_812C144.s [hunk 12]
```diff
@@ -383,7 +554,41 @@ gBattleAnimSpriteTemplate_8402738:: @ 8402738
 
 	.align 2
 gUnknown_08402750:: @ 8402750
-	.incbin "baserom.gba", 0x00402750, 0x30
+	.2byte 0x0010
```
Comment (marijnvdwerf) [inline data/battle_anim_812C144.s]: obj_rot_scal_anim_frame 16, 0, 0, 4
	obj_rot_scal_anim_frame 0, -3, 0, 16
	obj_rot_scal_anim_frame 4, 0, 0, 4
	obj_rot_scal_anim_frame 0, 0, 0, 24
	obj_rot_scal_anim_frame -5, 3, 0, 16
	obj_rot_scal_anim_end 0

File: data/battle_anim_812C144.s [hunk 13]
```diff
@@ -300,13 +446,37 @@ gBattleAnimSpriteTemplate_84025EC:: @ 84025EC
...
+	.4byte 0x00285078
 
+	.align 2
 gUnknown_08402608:: @ 8402608
 	.incbin "baserom.gba", 0x00402608, 0x8
 
+	.align 2
 gUnknown_08402610:: @ 8402610
-	.incbin "baserom.gba", 0x00402610, 0x20
+	.2byte 0x0000
```
Comment (marijnvdwerf) [inline data/battle_anim_812C144.s]: obj_rot_scal_anim_frame 0, -15, 0, 7
	obj_rot_scal_anim_frame 0, 15, 0, 7
	obj_rot_scal_anim_loop 2
	obj_rot_scal_anim_end 0

File: data/battle_anim_812C144.s [hunk 14]
```diff
@@ -254,23 +298,125 @@ gSpriteTemplate_8402500:: @ 8402500
...
+	.byte 0x00, 0x00 @ struct padding
 
 	.align 2
 gBattleAnimSpriteTemplate_8402578:: @ 8402578
 	spr_template 10236, 10236, gOamData_837DF24, gDummySpriteAnimTable, NULL, gDummySpriteAffineAnimTable, sub_812D724
 
 	.align 2
 gUnknown_08402590:: @ 8402590
-	.incbin "baserom.gba", 0x00402590, 0x30
+	.2byte 0x0000
```
Comment (marijnvdwerf) [inline data/battle_anim_812C144.s]: obj_rot_scal_anim_frame 0, 6, 0, 20
	obj_rot_scal_anim_frame 0, 0, 0, 20
	obj_rot_scal_anim_frame 7, -30, 0, 6
	obj_rot_scal_anim_frame 0, 0, 0, 20
	obj_rot_scal_anim_frame -2, 3, 0, 20
	obj_rot_scal_anim_end 0

File: data/battle_anim_812C144.s [hunk 15]
```diff
@@ -254,23 +298,125 @@ gSpriteTemplate_8402500:: @ 8402500
...
+	.2byte 0x7FFF
+	.2byte 0x0000
+	.byte 0x00
+	.byte 0x00
+	.byte 0x00, 0x00 @ struct padding
 
 	.align 2
 gUnknown_08402540:: @ 8402540
-	.incbin "baserom.gba", 0x00402540, 0x38
+	.2byte 0x0000
```
Comment (marijnvdwerf) [inline data/battle_anim_812C144.s]: obj_rot_scal_anim_frame 0, 6, 0, 20
	obj_rot_scal_anim_frame 0, 0, 0, 20
	obj_rot_scal_anim_frame 0, -18, 0, 6
	obj_rot_scal_anim_frame -18, -18, 0, 3
	obj_rot_scal_anim_frame 0, 0, 0, 15
	obj_rot_scal_anim_frame 4, 4, 0, 13
	obj_rot_scal_anim_end 0

File: data/battle_anim_812C144.s [hunk 16]
```diff
@@ -254,23 +298,125 @@ gSpriteTemplate_8402500:: @ 8402500
 
 	.align 2
 gUnknown_08402518:: @ 8402518
-	.incbin "baserom.gba", 0x00402518, 0x28
+	.2byte 0x0008
```
Comment (marijnvdwerf) [inline data/battle_anim_812C144.s]: obj_rot_scal_anim_frame 8, -8, 0, 12
	obj_rot_scal_anim_frame -16, 16, 0, 12
	obj_rot_scal_anim_frame 8, -8, 0, 12
	obj_rot_scal_anim_loop 1
	obj_rot_scal_anim_end 0

File: data/battle_anim_812C144.s [hunk 17]
```diff
@@ -238,7 +260,29 @@ gBattleAnimSpriteTemplate_8402498:: @ 8402498
 
 	.align 2
 gUnknown_084024B0:: @ 84024B0
-	.incbin "baserom.gba", 0x004024b0, 0x20
+	.2byte 0xFFF4
```
Comment (marijnvdwerf) [inline data/battle_anim_812C144.s]: obj_rot_scal_anim_frame -12, 20, 0, 8
	obj_rot_scal_anim_frame 12, -20, 0, 8
	obj_rot_scal_anim_loop 2
	obj_rot_scal_anim_end 0

File: data/battle_anim_812C144.s [hunk 18]
```diff
@@ -189,7 +189,29 @@ gBattleAnimSpriteTemplate_84023E8:: @ 84023E8
 
 	.align 2
 gUnknown_08402400:: @ 8402400
-	.incbin "baserom.gba", 0x00402400, 0x20
+	.2byte 0xFFF4
```
Comment (marijnvdwerf) [inline data/battle_anim_812C144.s]: obj_rot_scal_anim_frame -12, 8, 0, 4
	obj_rot_scal_anim_frame 20, -20, 0, 4
	obj_rot_scal_anim_frame -8, 12, 0, 4
	obj_rot_scal_anim_end 0

File: data/battle_anim_80CA710.s [hunk 19]
```diff
@@ -3986,10 +4144,21 @@ gBattleAnimSpriteTemplate_83DACE8:: @ 83DACE8
 
 	.align 2
 Unknown_3DAD00:: @ 83DAD00
-	.incbin "baserom.gba", 0x3dad00, 0x10
+	.2byte 0xFFF0
```
Comment (marijnvdwerf) [inline data/battle_anim_80CA710.s]: subsprite -16,   0, 1,   0, 32x16
	subsprite  16,   0, 1,   8, 32x16

File: data/battle_anim_812C144.s [hunk 20]
```diff
@@ -254,23 +260,38 @@ gSpriteTemplate_8402500:: @ 8402500
...
+	obj_rot_scal_anim_frame 0, 0, 0, 20
+	obj_rot_scal_anim_frame 7, -30, 0, 6
+	obj_rot_scal_anim_frame 0, 0, 0, 20
+	obj_rot_scal_anim_frame -2, 3, 0, 20
+	obj_rot_scal_anim_end 0
 
 	.align 2
 gUnknown_084025C0:: @ 84025C0
-	.incbin "baserom.gba", 0x004025c0, 0x4
+	.4byte 0x00FC18E8
```
Comment (YamaArashi) [inline data/battle_anim_812C144.s]: byte array

File: data/battle_anim_812C144.s [hunk 21]
```diff
@@ -300,13 +321,18 @@ gBattleAnimSpriteTemplate_84025EC:: @ 84025EC
 
 	.align 2
 gUnknown_08402604:: @ 8402604
-	.incbin "baserom.gba", 0x00402604, 0x4
+	.4byte 0x00285078
```
Comment (YamaArashi) [inline data/battle_anim_812C144.s]: byte array

File: data/battle_anim_812C144.s (hunk 20 above)
Comment (ProjectRevoTPP) [inline data/battle_anim_812C144.s]:
```
_0812DD7C:
	ldr r2, _0812DDA8 @ =gUnknown_030042C0
	ldr r1, _0812DDAC @ =gUnknown_084025C0
	movs r4, 0xC
	ldrsh r0, [r3, r4]
	adds r0, r1
```

Comment (marijnvdwerf) [inline data/battle_anim_812C144.s]: followed by `ldrb r0, [r0]`

**Pull request merged**